workspaces/adventure-pack/goodies/typescript/Iterator.prototype.flatMap/index.test.ts

```diff
@@ -109,15 +109,20 @@ describe("Iterator.prototype.flatMap", () => {
       c: string,
       index: number,
     ): Generator<string, void, void> {
-      yield `${index}: ${c}`;
+      yield `${index}`;
+      yield `${c.repeat(index + 1)}`;
```

Comment:
This should be identical?

```suggestion
      yield c.repeat(index + 1);
```